core/services/fluxmonitor/flux_monitor.go

```diff
@@ -544,8 +542,14 @@ func (p *PollingDeviationChecker) consume() {
 	}
 
 	// subscribe to contract logs
-	isConnected, unsubscribeFALogs := p.fluxAggregator.SubscribeToLogs(p)
-	defer unsubscribeFALogs()
+	isConnected := false
+	fluxAggLogListener, err := newFluxAggregatorDecodingLogListener(p.fluxAggregator.Address(), p.store.EthClient, p)
+	if err != nil {
+		logger.Errorw("unable to create flux agg decoding log listener", "err", err)
```

Comment:
Yeah this is a good question. Let me look into it...

Reply:
So the only failure case is if it can't parse the json abi, which is generated so I think the risk is quite low. Changed this to return after the log so we don't try to continue running the job without a log listener.